admin/load.php

```diff
@@ -156,17 +156,48 @@ function perflab_render_modules_page_field( $module_slug, $module_data, $module_
 				<input type="checkbox" id="<?php echo esc_attr( "{$base_id}_enabled" ); ?>" aria-describedby="<?php echo esc_attr( "{$base_id}_description" ); ?>" disabled>
 				<input type="hidden" name="<?php echo esc_attr( "{$base_name}[enabled]" ); ?>" value="<?php echo $enabled ? '1' : '0'; ?>">
 				<?php
+				if ( 'database/sqlite' === $module_slug && file_exists( WP_CONTENT_DIR . '/db.php' ) && ! defined( 'PERFLAB_SQLITE_DB_DROPIN_VERSION' ) ) {
+					printf(
+						/* translators: %s: WP_CONTENT_DIR */
+						esc_html__( 'The SQLite module cannot be activated because a different %s drop-in already exists.', 'performance-lab' ),
+						'<code>' . esc_html( WP_CONTENT_DIR ) . '/db.php</code>'
+					);
+				} elseif ( 'database/sqlite' === $module_slug && ! wp_is_writable( WP_CONTENT_DIR ) ) {
+					printf(
+						/* translators: %s: WP_CONTENT_DIR */
+						esc_html__( 'The SQLite module cannot be activated because the %s directory is not writable.', 'performance-lab' ),
+						'<code>' . esc_html( WP_CONTENT_DIR ) . '</code>'
+					);
```

Comment:
No need to show the entire directory path. Also it's worth clarifying the translator comment which is misleading.
```suggestion
					printf(
						/* translators: %s: db.php drop-in path */
						esc_html__( 'The SQLite module cannot be activated because a different %s drop-in already exists.', 'performance-lab' ),
						'<code>' . esc_html( basename( WP_CONTENT_DIR ) ) . '/db.php</code>'
					);
				} elseif ( 'database/sqlite' === $module_slug && ! wp_is_writable( WP_CONTENT_DIR ) ) {
					printf(
						/* translators: %s: db.php drop-in path */
						esc_html__( 'The SQLite module cannot be activated because the %s directory is not writable.', 'performance-lab' ),
						'<code>' . esc_html( basename( WP_CONTENT_DIR ) ) . '</code>'
					);
```